websites/G/galaxy/presence.ts

```diff
@@ -0,0 +1,442 @@
+				{
+					label: "Search games",
+					url: href,
+				},
+			];
+			break;
+		}
+		case "explore": {
+			const categories: Record<string, string> = {
+				top: "Browsing the top-rated games",
+				faves: "Browsing the most favorited games",
+				recent: "Browsing the most recently updated games",
+				playtime: "Browsing the most grinded games",
+				new: "Browsing the newest games",
+				random: "Browsing random games",
+			};
+			presenceData.details = categories[path[1]] || "Browsing for games";
+			presenceData.buttons = [
+				{
+					label: "Explore games",
+					url: href,
+				},
+			];
+			break;
+		}
+		// Other pages
+		case "signup": {
+			presenceData.details = "Signing up";
+			break;
+		}
+		case "request": {
+			const gameName = document.querySelector<HTMLInputElement>(
+				'div[data-label="game name"] > input'
+			)?.value;
+
+			presenceData.details = "Requesting a game";
+			if (gameName) presenceData.state = `"${gameName}"`;
+			presenceData.buttons = [
+				{
+					label: "Request game",
+					url: href,
+				},
+			];
+			break;
+		}
+		case "stats": {
+			presenceData.details = "Viewing site statistics";
+			presenceData.buttons = [
+				{
+					label: "View stats",
+					url: href,
+				},
+			];
+			break;
+		}
+		case "cluster": {
+			presenceData.details = "Trying out galaxy cluster";
+			presenceData.buttons = [
+				{
+					label: "Galaxy cluster",
+					url: href,
+				},
+			];
+			break;
+		}
+		case "dev": {
+			presenceData.details = "Becoming a developer";
+			presenceData.buttons = [
+				{
+					label: "View page",
+					url: href,
+				},
+			];
+			break;
+		}
+		case "docs": {
+			presenceData.details = "Reading the documentation";
+			presenceData.buttons = [
+				{
+					label: "Read the docs",
+					url: href,
+				},
+			];
+			break;
+		}
+		case "about": {
+			presenceData.details = "Reading about galaxy";
+			presenceData.buttons = [
+				{
+					label: "View page",
+					url: href,
+				},
+			];
+			break;
+		}
+		case "rules": {
+			presenceData.details = "Reviewing the site rules";
+			presenceData.buttons = [
+				{
+					label: "View page",
+					url: href,
+				},
+			];
+			break;
+		}
+		case "terms": {
+			presenceData.details = "Reading the terms of service";
+			presenceData.buttons = [
+				{
+					label: "View page",
+					url: href,
+				},
+			];
+			break;
+		}
+		case "privacy": {
+			presenceData.details = "Reading the privacy policy";
+			presenceData.buttons = [
+				{
+					label: "View page",
+					url: href,
+				},
+			];
+			break;
+		}
+		case "cookies": {
+			presenceData.details = "Reading the cookie policy";
+			presenceData.buttons = [
+				{
+					label: "View page",
+					url: href,
+				},
+			];
+			break;
+		}
+		case "banned": {
+			presenceData.details = "Banned";
+			break;
+		}
+		case "pogger": {
+			// https://galaxy.click/pogger
+			presenceData.buttons = [
+				{
+					label: "pogger button",
+					url: "https://save418.com",
+				},
+			];
```

Comment:
Please remove, we don't really allow static buttons as they can be seen as advertisements

Reply:
I changed it to the `href` of the page the button appears on